docs/get-started.md

```diff
@@ -319,7 +319,7 @@ Q3. Find the selected courses and corresponding grades of students liked by 201.
 
 ```
 -- By pipe
-nebula> GO FROM 201 OVER like | GO FROM $-.id OVER select YIELD $^.student.name AS Student, $$.course.name AS Course, select.grade AS Grade;
+nebula> GO FROM 201 OVER like yield like._dst as id | GO FROM $-.id OVER select YIELD $^.student.name AS Student, $$.course.name AS Course, select.grade AS Grade;
```

Comment:
![1](https://user-images.githubusercontent.com/51590253/65243794-f3695d00-da9d-11e9-9df5-a1171dfc1344.jpg)

However, the original statement is not working either.
![1](https://user-images.githubusercontent.com/51590253/65243982-64a91000-da9e-11e9-8a47-25c16a02a914.jpg)